src/components/pages/home/jumbotron/index.tsx

```diff
@@ -61,7 +67,9 @@ const Jumbotron: FunctionComponent<JumbotronProps> = ({resource}) => {
               </h2>
               <Link href={path}>
                 <a
-                  className="sm:text-2xl md:text-4xl text-xl max-w-screen-lg font-extrabold leading-tighter hover:text-yellow-500"
+                  className={`sm:text-2xl md:text-4xl text-xl max-w-screen-lg font-extrabold leading-tighter ${
+                    titleHover ? titleHover : 'hover:text-blue-600'
+                  }`}
```

Comment:
Added a `titleHover` Prop because we change the hardcoded with value with every update to the Jumbotron. Since we are starting to use the Jumbotron elsewhere, it make sense to make this dynamic